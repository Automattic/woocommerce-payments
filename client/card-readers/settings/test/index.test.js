/**
 * External dependencies
 */
import { render, screen } from '@testing-library/react';

/**
 * Internal dependencies
 */
import ReceiptSettings from '..';

describe( 'ReceiptSettings', () => {
	test( 'Readers merchant settings page renders', () => {
		render( <ReceiptSettings /> );

		expect(
			screen.queryByText( 'Card reader receipts' )
		).toBeInTheDocument();
	} );

	test( 'Readers merchant settings page snapshot test', () => {
		const { container } = render( <ReceiptSettings /> );
		expect( container ).toMatchSnapshot();
	} );
} );
