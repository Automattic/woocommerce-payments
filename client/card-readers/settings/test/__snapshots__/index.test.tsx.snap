// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReceiptSettings Readers merchant settings page snapshot test 1`] = `
<div>
  <div
    class="wcpay-settings-layout"
  >
    <div
      class="settings-section"
    >
      <div
        class="settings-section__details"
      >
        <h2>
          Card reader receipts
        </h2>
        <p>
          These details will appear on printed or emailed receipts for customers that pay in person using card readers. Updating the details here will not affect any other stores settings.
        </p>
        <a
          data-link-type="wc-admin"
          href="admin.php?page=wc-admin&path=%2Fpayments%2Fcard-readers%2Fpreview-receipt"
        >
          Preview a printed receipt
        </a>
      </div>
      <div
        class="settings-section__controls"
      >
        <span
          aria-busy="true"
          class="is-loadable-placeholder is-block"
        >
          <p
            style="line-height: 20;"
          >
            Block placeholder
          </p>
        </span>
      </div>
    </div>
    <div
      class="settings-section save-settings-section"
    >
      <div
        class="settings-section__details"
      />
      <div
        class="settings-section__controls"
      >
        <button
          class="components-button is-primary"
          disabled=""
          type="button"
        >
          Save changes
        </button>
      </div>
    </div>
  </div>
</div>
`;
